projects/plugins/boost/app/assets/src/js/global.d.ts

```diff
@@ -48,6 +48,13 @@ declare global {
 		nonces: {
 			[ key: string ]: string;
 		};
+		pricing: {
```

Comment:
This type doesn't match what `add_js_constants` produces.

Reply:
I didn't get that. Do you mean the possibility of `yearly` being empty?

Reply:
Yes. We're not ensuring that `pricing` has any other values than `yearly`:

https://github.com/Automattic/jetpack/pull/24503/files#diff-d7bb61f231c4b097b8493b260a1dbd6e7d6b82b4d4972428a515efd3fdf03713R16-R18

Because it uses `Wpcom_Products::get_product_pricing` under the hood, and that function can return an empty array for example. Also because it's a function that we inherit via a package, we should also be prepared for a situation where the return format may change.

Reply:
Makes sense. I have updated the type and how the constant is passed.